aten/src/ATen/native/mkldnn/xpu/Blas.cpp

```diff
@@ -631,4 +631,175 @@ Tensor _weight_int8pack_mm_xpu(
 
   return C;
 }
+
+Tensor _bmm_dtype_xpu(
+    const Tensor& batch1,
+    const Tensor& batch2,
+    const at::ScalarType out_dtype) {
+  Tensor out = at::empty(
+      {batch1.size(0), batch1.size(1), batch2.size(2)},
+      batch1.options().dtype(out_dtype));
+  return _bmm_out_dtype_xpu(batch1, batch2, out_dtype, out);
+}
+
+Tensor& _bmm_out_dtype_xpu(
+    const Tensor& batch1,
+    const Tensor& batch2,
+    const at::ScalarType out_dtype,
```

Comment:
the out_dtype is not used here?

Reply:
Hi, @gujinghui  this is to match the function schema defined in native_functions.yaml.
And I've updated the code which we need to check the out_dtype here, thanks for the reminder!